data/drak/indigenous.txt

```diff
@@ -249,6 +249,72 @@ ship "Embershade"
 
 
 
+ship "Embersylph"
+	noun "creature"
+	sprite "ship/embersylph/embersylph"
+		"frame rate" 36
+		"random start frame"
+		"scale" 0.5
+	"never disabled"
+	attributes
+		category "Light Warship"
+		"shields" 7200
+		"hull" 1800
+		"required crew" 1
+		"bunks" 1
+		"mass" 140
+		"drag" 3.2
+		"heat dissipation" .7
+		"cargo space" 0
+		"fuel capacity" 100
+		"thrust" 8
+		"reverse thrust" 3
+		"turn" 90
+		"energy capacity" 800
+		"energy generation" 7
+		"hull repair rate" 2
+		"shield generation" 2
+		"scram drive" 0.4
+		"jump fuel" 100
+		"hyperdrive" 1
+		"ramscoop" 1.5
+		"gaslining" 1
+		"inscrutable" 1
+		"atmospheric scan" 1000
+		"shield resistance" 1
+	outfits
+		"Absorption Organ" 1
+	turret 0 0 "Absorption Organ"
+	leak "blood" 2
+	explode "blood" 1
+	description "Ember Ray."
+	
+ship "Embersylph" "Embersylph (Small)"
+	sprite "ship/embersylph/embersylph"
+		"scale" 0.5
+	add attributes
+		"shields" -2800
+		"hull" -900
+		"thrust" 2
```

Comment:
Unless you want it to be possible to disable this one, you need to give it `never disabled` as well.